lib/nats.js

```diff
@@ -547,6 +595,8 @@ Client.prototype.createConnection = function() {
     }
     // Create the stream
     this.stream = net.createConnection(this.url.port, this.url.hostname);
+    this.stream.id = nuid.next();
+    this.emit('created', this.stream.id);
```

Comment:
Do you need those 2 lines? I don't see the stream's 'created' event being used in the new test.